src/shared/icons/styles.scss

```diff
@@ -0,0 +1,8 @@
+.icon { //need classname
+	width: 128px;
+	height: 128px;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
```

Comment:
This is specific to the `<EpisodePreview />` component - so shouldn't be defined here, and should be passed down via the `className` prop that I introduced up there ^^

Reply:
cool, makes sense! :)